src/components/organisms/cards/cards.html

```diff
@@ -0,0 +1,49 @@
+<div class="organisms-cards">
```

Comment:
```suggestion
<div class="organisms-cards container">
```
`.container` で横幅を管理するとこういう感じになります！